apps/codeforafrica/src/lib/data/blockify/get-involved.js

```diff
@@ -0,0 +1,14 @@
+import { formatImpacts } from "./our-impact";
+
+function getInvolved(block) {
+  const { impacts, ...other } = block;
+  const ourImpacts = formatImpacts(impacts);
```

Comment:
I think, either:
1. Make `formatImpacts` or `impacts` a standalone function (i.e. in its own file) imported in both blocks, OR
2. Just import `ourImpact` and use it her by overriding `slug`.

----

In the back of our minds we should really think if these needs to be 2 separate blocks or one block can do.